plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotModule.java

```diff
@@ -120,6 +123,19 @@ public DataSchema deserialize(JsonParser p, DeserializationContext ctxt)
         }
     }
 
+    public static class BrokerResponseNativeDeserializer
```

Comment:
Note: this was added due changes to the json serialization in this Pinot [pr](https://github.com/apache/pinot/pull/13179).